load-testing/tests/relays_stress_helpers_test.go

```diff
@@ -631,7 +651,8 @@ func (s *relaysSuite) getAppFundingAmount(currentBlockHeight int64) sdk.Coin {
 	// based on the number of relays it needs to send. Theoretically, `+1` should
 	// be enough, but probabilistic and time based mechanisms make it hard
 	// to predict exactly.
-	appFundingAmount := s.relayRatePerApp * s.relayCoinAmountCost * currentTestDuration * blockDuration * 2
+	amountToSpend := s.relayRatePerApp * s.relayCoinAmountCost * currentTestDuration * blockDuration
+	appFundingAmount := math.Max(amountToSpend, s.appParams.MinStake.Amount.Int64()*2)
```

Comment:
Move the comment starting with `Multiply by 2` above so it's above this line instead.